class.application-passwords.php

```diff
@@ -568,11 +568,13 @@ public static function show_user_profile( $user ) {
 			'application-passwords-js',
 			'appPass',
 			array(
-				'root'      => esc_url_raw( rest_url() ),
-				'namespace' => '2fa/v1',
-				'nonce'     => wp_create_nonce( 'wp_rest' ),
-				'user_id'   => $user->ID,
-				'text'      => array(
+				'root'                 => esc_url_raw( rest_url() ),
+				'namespace'            => '2fa/v1',
+				'nonce'                => wp_create_nonce( 'wp_rest' ),
+				'user_id'              => $user->ID,
+				'revoke_password'      => esc_attr__( 'Are you sure you want to revoke this password? This action cannot be undone.' ),
```

Comment:
Maybe add these strings to the `text` array which already contains one for `no_credentials`?

Reply:
Nice catch! Let's do it! Thank you @kasparsd